File: src/atcoder/abc229/abc229_a.java
import java.util.Scanner;

public class abc229_a {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.next();
        String b = sc.next();
        sc.close();
        String ab = a + b;
        // System.out.println(ab);
        if (ab.equals(".##.") || ab.equals("#..#")) {
            System.out.println("No");
        } else {
            System.out.println("Yes");
        }
    }

}

File: src/atcoder/abc229/abc229_a.c
#include "stdio.h"
int main(void)
{
  char a[3], b[3];
  scanf("%s\n%s", a, b);
  if (a[0] != b[0] && a[1] != b[1])
  {
    if (a[0] == a[1] && b[0] == b[1])
    {
      printf("Yes");
    }
    else
    {
      printf("No");
    }
  }
  else
  {
    printf("Yes");
  }
  return 0;
}
